README.md

```diff
@@ -173,7 +173,7 @@ yarn ios:old
 
 Or by choosing Product > Run from Xcode, or by clicking the Run button in your project’s toolbar, similarly how you would run a native iOS application.
 
-> **_NOTE_**  
+> **_NOTE_**
```

Comment:
Please update these to the current GitHub format (see [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)):
```markdown
> [!NOTE]
> ...
```

Reply:
Fixed.